code/__DEFINES/antagonists.dm

```diff
@@ -275,7 +275,7 @@ GLOBAL_LIST_INIT(human_invader_antagonists, list(
 #define MAROON_PROB 30
 
 /// How many telecrystals a normal traitor starts with
-#define TELECRYSTALS_DEFAULT 20
+#define TELECRYSTALS_DEFAULT 30 //BUBBERSTATION CHANGE: 20 TO 30.
```

Comment:
Discard this and set it back down to 20. We have another test merge PR right now that sets it to 25.